package.json

```
@@ -22,22 +24,15 @@
     }
   },
   "dependencies": {
-    "case-police": "^0.5.14",
-    "eslint": "^8.20.0",
+    "eslint": "^8",
```

```suggestion
    "eslint": "^8.57.1",
```

Revert more granular range.